From fc7697e91973b81d6e6420fa287792ecf2301a96 Mon Sep 17 00:00:00 2001
From: Richard Laager <rlaager@wiktel.com>
Date: Tue, 10 May 2022 20:31:39 -0500
Subject: [PATCH] ntplogtemp: Fix printing blank line on exit

This was leading to cron mail to a user every 5 minutes.
---
 ntpclients/ntplogtemp.py | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/ntpclients/ntplogtemp.py b/ntpclients/ntplogtemp.py
index bacaef473..6d85d416c 100644
--- a/ntpclients/ntplogtemp.py
+++ b/ntpclients/ntplogtemp.py
@@ -107,7 +107,7 @@ class CpuTemp:
             self.has_sensors = False
             if args.verbose:
                 sys.stderr.write("No sensors returned temperatures. ",
-                                 "Have you run sensors-detect?")
+                                 "Have you run sensors-detect?\n")
 
         return _data
 
@@ -308,11 +308,10 @@ def log_data():
 
 args = parser.parse_args()
 if os.getuid():
-    sys.stderr.write("You must be root!")
+    sys.stderr.write("You must be root!\n")
     sys.exit(1)
 
 try:
     log_data()
-except (KeyboardInterrupt, SystemExit):
+except KeyboardInterrupt:
     print("")    # be nice to bash
-    sys.exit(0)
-- 
2.25.1

